app/static/main.css

```diff
@@ -144,3 +144,12 @@ footer{
   opacity: 0.8;
 }
 
+.btn{
```

Comment:
Causing a merge conflict. Class selector is too vague and will select all buttons on the document. Easy fix would be to use an id to select the buttons you want to style :)